gradle/maven-support.gradle

```diff
@@ -464,7 +467,25 @@ afterEvaluate { project ->
                 // FIXME :: @chrisdoyle :: this assumes Java 7...
                 options.links 'http://docs.oracle.com/javase/7/docs/api/'
                 options.links("http://developer.android.com/reference/");
+                if (isGraphvizDotAvailable()) {
 
+                    options {
+                        doclet = 'org.umlgraph.doclet.UmlGraphDoc'
+                        options.docletpath = configurations.umljavadoc.files.asType(List)
+                        addStringOption('-inferrel')
+                        addStringOption('-inferdep')
+                        addStringOption('-qualify')
+                        addStringOption('-postfixpackage')
+                        addStringOption('-hide=java.*')
+                        addStringOption('-collpackages=java.util.*')
+                        addStringOption('-nodefontsize=9')
+                        addStringOption('-nodefontpackagesize=7')
+                        addStringOption('-link=http://java.sun.com/j2se/1.5.0/docs/guide/javadoc/doclet/spec')
+                        addStringOption('-link=http://java.sun.com/j2se/1.5/docs/api')
```

Comment:
We may want to add support for newer JDK versions, maybe pass in a configuration with a default to whatever the latest least supported version is (i.e. 7)?